supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -1,6 +1,49 @@
 [[formatting]]
 = Formatting
 
+[[commands-in-code-blocks]]
+== Commands in code blocks
+
+Use a single command per code block for each procedure step. This helps with readability and makes it possible for the copy button in code blocks to work correctly.
+
+As a default, use bold formatting for commands in code blocks to visually distinguish them from their lead-in sentences and example outputs. For consistency, do this even when no output is shown in the code block.
+
+[NOTE]
+====
+To apply formatting in a code block, you must use the `quotes` link:https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/[AsciiDoc substitution].
+====
+
+.Example AsciiDoc: A command and its output in a code block
+
+  Verify that the `libvirt` default network is active and configured to start automatically:
+  
+  [subs="+quotes"]
+  ----
+  # *virsh net-list --all*
+  Name      State    Autostart   Persistent
+  --------------------------------------------
+  default   active   yes         yes
+  ----
+
+
+This example renders as follows in HTML:
+
+====
+Verify that the `libvirt` default network is active and configured to start automatically:
+
+[subs="+quotes"]
+----
+# *virsh net-list --all*
+Name      State    Autostart   Persistent
+--------------------------------------------
+default   active   yes         yes
+----
+====
+
+Do not use bold formatting for the output, unless you need to highlight a specific part of it. When you do emphasize a non-command item in a code block, you are not required to bold the command.
```

Comment:
I don't think this is good guidance. Writers should still be able to use bolded strings in outputs if necessary, especially if in the given code block, the output is more important than the command itself. Also, it would be rather strange to express the same kind of emphasis differently in the very same code block.

Your other suggestions are decidedly improvements, though, thank you :-)

Reply:
@jherrman Apologies for the delay in replying to your comment. It seems odd to me to say bold the command in one place and not in another, but perhaps I misunderstood the intent of this update.

Reply:
Well, the idea is basically "bold the important part, so that it's visually distinct" - and since the command itself is usually the important part of a CLI example (and the given step in general), it would be the default part do bold. However,  when a part of the output is as important as the command in the given context, or more, we should not bar the writer from bolding something else than the command, and potentially even leaving the command un-bolded.

It's entirely possible I did not formulate that clearly enough in the actual source update, though...

Reply:
@jafiala Can you give us an example of what you're thinking of with bolding pieces of the output?

Reply:
I suspect that @ was meant to aim at me :-) A few examples I fished out:

- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/configuring_and_managing_virtualization/index#setting-up-secure-execution-on-ibm-z_securing-virtual-machines-in-rhel -> Verification (highlights where in the output the user can find the line they should look for)
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/configuring_and_managing_virtualization/index#setting-up-secure-execution-on-ibm-z_securing-virtual-machines-in-rhel -> Prerequisites and Step 8i (again, highlighting what the user should look for)

I probably don't have examples with bolded output and un-bolded commands, but if in the context you really need to highlight a part of the output rather than the command  you use to get it, I don't think we should prohibit that, either.

Reply:
Thanks for the examples, @jherrman. 

I think it's better to highlight such lines in the command output by using a callout instead of bold text? The following example shows callouts (in a code example, but the same method could be used for command output):
https://docs.openshift.com/pipelines/1.13/about/understanding-openshift-pipelines.html

WDYT?

Reply:
Honestly, callouts do not seem like an optimal solution to me.

- They are pretty wonky to work with in adoc (for instance can only be at the end of a line, cannot be used out of sequence, one callout cannot be used repeatedly, and they must always have an explanation under the codeblock, AFAIK). Also, Customer Portal stylesheets do not render them very well, currently.

- When they work as expected, callouts are visually very distinct, which may not always be what you want when highlighting a small part of a command output. 

If the general intention behind this update is "bold out the operative/important parts of the code block for greater visual clarity", I would say the output should follow that consistently with the commands. Also, it's easier for writers to internalize and follow.

Reply:
@jherrman and @dfitzmau Thank you both! I understand that callouts might not be optimal, but I wonder if we could highlight the command output in some other way rather than by reusing the bold format. 

Do you know whether the highlight syntax described [here](https://docs.asciidoctor.org/asciidoc/latest/text/highlight/) would render correctly in Customer Portal and OpenShift docs?

Reply:
I tried that syntax a few years ago on the CP and it did render, but I was informed to drop it as the yellow highlighting was really distracting. It definetly grabs attention, but it distracts from the surrounding content. I can understand why yellow is a well-adopted colour for safety :yellow_heart:

Reply:
I tested out the `#highlight#` markup earlier today. Renders reasonably well in asciidoctor (though, as Darragh points out, the colour is a bit much), but both bccutil and the Customer Portal stylesheets completely ignore it, unfortunately: 

https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/zen_and_zany_asciidoc_training_titles/42/html-single/davozenis_advanced_asciidoc_training/index?lb_target=stage#proc_test-two_assembly_nested-test

Source:

> [subs="+quotes"]
> ----
> # *Sample command sting*
>
> Sample #output# string
----